common/men_at_arms_types/temp_draft_doc.txt

```diff
@@ -0,0 +1,575 @@
+﻿@cultural_maa_extra_ai_score = 60 # Equivalent to having 6 extra regiments beyond what the code scoring would indicate (see NEGATIVE_SCORE_PER_EXISTING_REGIMENT)
+#Centuar
+deathcaller = {
```

Comment:
You declared all your MaAs twice.

Reply:
Wym? I only see deathcaller twice, but I will replace everything in the main folder with everything from the temp folder when you say you agree with all maa. Keep them in the temp so its easier for reviewers to see

Reply:
> Wym? I only see deathcaller twice, but I will replace everything in the main folder with everything from the temp folder when you say you agree with all maa. I put them in the temp so its easier for reviewers to see

Where you said it to testers? The PR desc has nothing. I was supposed to review this file?
Why you have temp files if you requested my review. There shouldn't be temp files if you intent to merge it to the master.